jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/web/action/ScrollOnWebAction.java

```diff
@@ -0,0 +1,23 @@
+package org.jbehavesupport.core.internal.web.action;
+
+import org.jbehavesupport.core.web.WebActionContext;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ScrollOnWebAction extends AbstractWebAction {
+
+    @Override
+    public String name() {
+        return "SCROLL_ON";
```

Comment:
I'd keep original selenium wording, `MOVE_TO`, to simplified understanding what it really do.